content/en/docs/instrumentation/ruby/sampling.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Sampling
+weight: 6
+---
+
+[Sampling](/docs/concepts/sampling/) is a process that restricts the amount of
+traces that are generated by a system. The Ruby SDK offers several
+[head samplers](/docs/concepts/sampling#head-sampling).
+
+## Default behavior
+
+By default, all spans are sampled, and thus, 100% of traces are sampled. If you
+do not need to manage data volume, don't bother setting a sampler.
+
+## TraceIDRatioBased Sampler
+
+The most common head sampler to use is the [TraceIdRatioBased][] sampler. It
+deterministically samples a percentage of traces that you pass in as a
+parameter.
+
+[TraceIdRatioBased]:
+  https://open-telemetry.github.io/opentelemetry-ruby/opentelemetry-sdk/latest/OpenTelemetry/SDK/Trace/Samplers/TraceIdRatioBased.html
```

Comment:
@chalin for some reason this url gives a 404 in CI. But the site is it points to is live and works?

Reply:
That sometimes happens when there are too many redirects or the redirects introduce too much latency. I'm investigating.

Could we get #2504 reviewed and merged so that the Netlify build can pass even if there are link-check issues (as is the case here)?

Reply:
Nope, in this case the Ruby docs are _actually_ returning a 404, then there's a script in the page that dynamically changes the location to point to the current version. It's probably not a good idea to return 404 in that case.

Reply:
FYI, I've filed the following issue:

- https://github.com/open-telemetry/opentelemetry-ruby/issues/1435